src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -122,6 +127,21 @@
      */
     private int authCacheTimeoutSeconds;
 
+    /**
+     * Certain operations can be spread to run asynchronously to gain a performance boost.  Instead of having
+     * each such task create its own threadpool, this property is used to create a globally accessible pool that should
+     * be used by all such operations.
+     *
```

Comment:
If you want a paragraph break in a javadoc, you need a `<p>`
```suggestion
     * <p>
```